src/main/java/racingcar/view/Validator.java

```diff
@@ -0,0 +1,34 @@
+package racingcar.view;
+
+import static racingcar.common.SystemMessage.CAR_NAME_DUPLICATE_ERROR;
+import static racingcar.common.SystemMessage.CAR_NAME_LENGTH_LIMIT_ERROR;
+
+import java.util.List;
+import racingcar.common.exception.InputValidationException;
+
+public class Validator {
+
+    private static final int LENGTH_LIMIT_MAX = 5;
+    private static final int LENGTH_LIMIT_MIN = 1;
+
+    private Validator() {
+    }
+
+    protected static void validateLengthLimit(final List<String> splitUserInput) {
+        boolean match = splitUserInput.stream().allMatch(
+            input -> input.length() <= LENGTH_LIMIT_MAX && input.length() >= LENGTH_LIMIT_MIN);
```

Comment:
```suggestion
            input -> nameLength <= LENGTH_LIMIT_MAX && nameLength >= LENGTH_LIMIT_MIN);
```
length()를 두번 호출하는 것보다 변수로 할당해 한번만 호출하도록 수정해보는 것은 어떨까요?